tests/test_election_settings.py

```diff
@@ -0,0 +1,79 @@
+import json
+import pytest
+from flask.testing import FlaskClient
+
+from arlo_server import app, db
+from arlo_server.models import Election
+
+from tests.helpers import post_json, put_json
+
+
+@pytest.fixture
+def client() -> FlaskClient:
+    app.config["TESTING"] = True
+    client = app.test_client()
+
+    with app.app_context():
+        db.drop_all()
+        db.create_all()
+
+    yield client
+
+    db.session.commit()
+
+
+@pytest.fixture
+def election_id(client: FlaskClient, request) -> str:
+    rv = post_json(
+        client,
+        "/election/new",
+        {"auditName": f"Test Audit {request.function.__name__}"},
+    )
+    election_id = json.loads(rv.data)["electionId"]
+    yield election_id
+
+
+def test_update_election(client: FlaskClient, election_id: str):
+    rv = put_json(
+        client,
+        f"/election/{election_id}/settings",
+        {
+            "electionName": "An Updated Name",
+            "online": False,
+            "riskLimit": None,
+            "randomSeed": None,
+            "state": None,
+        },
+    )
+    assert rv.status_code == 200, f"unexpected response: {rv.data}"
+    assert json.loads(rv.data) == {"status": "ok"}
+    assert Election.query.filter_by(id=election_id).one().name == "An Updated Name"
+
+
+def test_get_empty(client: FlaskClient, election_id: str):
+    rv = client.get(f"/election/{election_id}/settings")
+    assert rv.status_code == 200, f"unexpected response: {rv.data}"
+    assert json.loads(rv.data) == {
+        "electionName": "",
+        "online": False,
+        "randomSeed": None,
+        "riskLimit": None,
+        "state": None,
+    }
+
+
+def test_update_election_with_invalid_property(client: FlaskClient, election_id: str):
+    rv = put_json(
+        client,
+        f"/election/{election_id}/settings",
+        {"electionNameTypo": "An Updated Name"},
+    )
+    assert rv.status_code == 400, f"unexpected response: {rv.data}"
+    assert json.loads(rv.data) == {
+        "errors": [
+            {
+                "message": "Additional properties are not allowed ('electionNameTypo' was unexpected)",
+                "errorType": "Bad Request",
+            }
+        ]
+    }
```

Comment:
Do you need a test for invalid values as well?

Reply:
Sure, will do.